elasticdl/proto/elasticdl.proto

```diff
@@ -160,3 +168,32 @@ service Pserver {
   rpc push_embedding_info(Model) returns (google.protobuf.Empty);
   rpc push_gradient(PushGradientRequest) returns (PushGradientResponse);
 }
+
+message PullDenseParametersRequest {
+  int32 version = 1;
+}
+
+message PullDenseParametersResponse {
+  bool initialized = 1;
+  int32 version = 2;
+  map<string, Tensor> dense_parameters = 3;
+}
+
+message PullEmbeddingTableRequest {
+  string name = 1;
+  repeated int64 ids = 2;
+}
+
+message PushGradientsResponse {
+  bool accepted = 1;
+  int32 version = 2;
+}
+
+// GoPS service
+service GoPserver {
```

Comment:
The service is independent with programming languages.

Reply:
Done. Remove it.

And add the new rpc service described in concepts doc into current Pserver service. After the refactoring work is done, old rpc service will be remove safely.